src/routes/auth/+page.svelte

```diff
@@ -285,6 +285,7 @@
       {options}
       required
       class="text-sm rounded-r-lg block w-full p-2.5 focus:outline-none border border-sky-300 border-l-sky-100 text-gray-900"
+      autocomplete="one-time-code"
```

Comment:
should be set to `tel` to auto-complete with the phone number

Reply:
ah. yeah.
sorry. 
will fix it soon.